src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState, useEffect } from "react";
+import convertTimeStamp from "../utils/convertTimeStamp";
+import heart from "../assets/heart-fill.png"
+import comment from "../assets/comment.png"
+
+export default function ArticleCard(article) {
+  const [title, setTitle] = useState("");
+  const [author, setAuthor] = useState("");
+  const [date, setDate] = useState("");
+  const [votes, setVotes] = useState("");
+  const [commentCount, setCommentCount] = useState("");
+  const [imgUrl, setImgUrl] = useState("");
+
+  useEffect(() => {
+    setTitle(article.article.title);
+    setAuthor(article.article.author);
+    setDate(convertTimeStamp(article.article.created_at));
+    setVotes(article.article.votes);
+    setCommentCount(article.article.comment_count);
+    setImgUrl(article.article.article_img_url);
+  }, []);
+
+  return (
+    <>
+      <div className="articleCardGrid">
```

Comment:
We could use a bit more semantic HTML in here. What tags might we use instead of div?